test/e2e/source_apiserver_test.go

```diff
@@ -88,11 +91,28 @@ func (test *e2eTest) setupServiceAccountForApiserver(t *testing.T, name string)
 	}
 	_, err = kubectl.RunWithOpts([]string{"create", "clusterrole", "testsa-role", "--verb=get,list,watch", "--resource=events,namespaces"}, runOpts{})
 	if err != nil {
-		t.Fatalf(fmt.Sprintf("Error executing 'kubectl clusterrole testsa-role'. Error: %s", err.Error()))
+		t.Fatalf(fmt.Sprintf("Error executing 'kubectl create clusterrole testsa-role'. Error: %s", err.Error()))
 	}
 	_, err = kubectl.RunWithOpts([]string{"create", "clusterrolebinding", "testsa-binding", "--clusterrole=testsa-role", "--serviceaccount=" + test.kn.namespace + ":" + name}, runOpts{})
 	if err != nil {
-		t.Fatalf(fmt.Sprintf("Error executing 'kubectl clusterrolebinding testsa-binding'. Error: %s", err.Error()))
+		t.Fatalf(fmt.Sprintf("Error executing 'kubectl create clusterrolebinding testsa-binding'. Error: %s", err.Error()))
+	}
+}
+
+func (test *e2eTest) deleteServiceAccountForApiserver(t *testing.T, name string) {
```

Comment:
This method is deleting ClusterRole and ClusterRoleBinding as well, IMO `tearDownForSourceApiServerTests` suites better for its name (similarly `setupServiceAccountForApiserver` in this file can be renamed to `setupForSourceApiServerTests`).

Reply:
> I've added a few suggestions, can be addressed in a subsequent PR as well.

Yes, I will open a subsequent  pr.

Reply:
#597